kubespawner/spawner.py

```diff
@@ -274,6 +291,32 @@ def _hub_connect_port_default(self):
         """
     )
 
+    storage = Unicode(
+        "1Gi",
+        config=True,
+        help="""
+        The ammount of storage space to request from the volume that the pvc will
+        mount to.
+        """
+    )
+
+    storage_class = Unicode(
+        "",
+        config=True,
+        help="""
+        The storage class that the pvc will use. If left blank, the kubespawner will not
+        create a pvc for the pod.
+        """
+    )
+
+    access_modes = List(
```

Comment:
The docs should also be self explanatory, and define the options. Can be just a short blurb + a link to the k8s docs. Can you expand help= for all fo the three properties?